spin.toml

```diff
@@ -1,14 +1,19 @@
-spin_version = "1"
-authors = ["Fermyon Engineering <engineering@fermyon.com>"]
-description = "A Favicon server written in (Tiny)Go"
-name = "spin-favicon"
-trigger = { type = "http", base = "/" }
-version = "1.0.0"
+spin_manifest_version = 2
 
-[[component]]
-id = "favicon"
+[application]
+name = "favicon"
+version = "0.2.0"
+authors = ["Matt Butcher <matt.butcher@fermyon.com>"]
+description = "Favicon server"
+
+[[trigger.http]]
+route = "/..."
+component = "favicon"
+
+[component.favicon]
 source = "favicon.wasm"
-files = ["favicon.ico"]
-[component.trigger]
-route = "/favicon.ico"
-executor = { type = "wagi" }
\ No newline at end of file
+# files = ["favicon.ico"]
```

Comment:
Can remove?

Reply:
Not sure yet. This is the part that seems broken. Neither syntax seems to work. I'll need to ask @adamreese

Reply:
@adamreese could you take a look at this?  Any go SDK updates needed to get the favicon server to work?